interop-testing/src/main/proto/io/grpc/testing/integration/empty.proto

```diff
@@ -30,7 +30,7 @@
 
 syntax = "proto2";
 
-package grpc.testing;
+package grpc.testing.integration;
```

Comment:
This would also require updating the service behind grpc-test.sandbox.googleapis.com.